src/main/java/GoogleApiClient.java

```diff
@@ -0,0 +1,50 @@
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Created by time2die on 20.11.16.
+ */
+public class GoogleApiClient {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.